pkg/operatorstatus/operator_status.go

```diff
@@ -209,6 +209,8 @@ func (r *ClusterOperatorStatusClient) RelatedObjects() []configv1.ObjectReferenc
 		{Group: "cluster.x-k8s.io", Resource: "clusters", Namespace: r.ManagedNamespace},
 		{Group: "cluster.x-k8s.io", Resource: "machines", Namespace: r.ManagedNamespace},
 		{Group: "cluster.x-k8s.io", Resource: "machinesets", Namespace: r.ManagedNamespace},
+		{Group: "admissionregistration", Resource: "validatingadmissionpolicy"},
+		{Group: "admissionregistration", Resource: "validatingadmissionpolicybinding"},
```

Comment:
It might be best to specify the resource names instead of catching all the VAP(B) in the cluster?

Reply:
I've updated with explicitly set names now, let's tackle this when we add more / in another PR :)